SPECS/systemd/systemd.signatures.json

```diff
@@ -1,5 +1,7 @@
 {
  "Signatures": {
+  "add-fsync-sysusers-passwd.patch": "c86c20c9bf0d79faeb99c454e8fb79df1b81d9243f45a209f7c8f58924ec7dba",
+  "gpt-auto-devno-not-determined.patch": "b75ee0a9b0e4f978d039cfbe9e5c9fd9314f99c3cd5e29f0d79752f9fc628c0d",
```

Comment:
Same comment here - no need to add the signatures.

Reply:
Removed